src/modules/utils.js

```diff
@@ -280,6 +280,14 @@ self = module.exports = {
             !url.href.match(/^https?:\/\/|^file:\/\//);
 
         return !!sameOrigin;
+    },
+
+    isNumber: function (value) {
+        if (typeof value === 'number') {
+            return true;
+        }
+
+        return !!value.match(/-?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?/);
```

Comment:
Agree that we can do better here, and support commas too. I took this regex from the validation that chrome does when a value is not a number, after a failure.  But supporting different notations, like european style, depends on i18n and l10n support that we don't have yet . We should add the support to all modules and plugins.